src/backends/glvisualize.jl

```diff
@@ -51,7 +51,8 @@ const _glvisualize_seriestype = [
 const _glvisualize_style = [:auto, :solid, :dash, :dot, :dashdot]
 const _glvisualize_marker = _allMarkers
 const _glvisualize_scale = [:identity, :ln, :log2, :log10]
-is_marker_supported(::GLVisualizeBackend, shape::Shape) = true
```

Comment:
Will `Plots.Shape` be supported eventually?  It's just a wrapper around 2D coordinate arrays.